docs/src/pages/docs/functions/logic-functions.md

```diff
@@ -308,6 +470,74 @@ function. Either the original or modified value will be returned depending on
 the result of the predicate. Check out [`unless`](#unless) for a negated version
 of this function.
 
+<!-- eslint-disable no-console -->
+
+```javascript
+import when from 'crocks/logic/when'
+
+import compose from 'crocks/helpers/compose'
+import composeB from 'crocks/combinators/composeB'
+import constant from 'crocks/combinators/constant'
+import curry from 'crocks/helpers/curry'
+import flip from 'crocks/combinators/flip'
+import mapProps from 'crocks/helpers/mapProps'
+
+// double :: Number -> Number
+const double = a =>
+  a * 2
+
+// doubleWhen :: (a -> Boolean) -> Number -> Number
+const doubleWhen =
```

Comment:
it's a little jarring for me to have the first example out the gate flip the function's params. I have to mentally work backwards to understand what when is from the example.

Reply:
Okay. I took the bottom bit of functions and moved them upstairs and added a some additional calls to `protectedSum` so it could be seen in action before working on a value on an object. I think should remove some of the cognitive load as a learner would experience? Providing a nice step up in complexity.

I am going to merge this as it is getting stale, but if you would like more work around this. We can def submit a new PR.